### PR TITLE
Add a FeatureType for enabling or disabling sending a notification when a case is confirmed

### LINKED ISSUE #4732: Individualize the notifications sent to users responsible for an event when an event participant becomes a case
### Situation Description
Development # 3794 must be extended with additional development.
To date, this notification is integrated into all "other notifications". In France, we do not want to activate all the other notifications since this generates too many emails (change of status of a case, new associated contact ...)

### Feature Description
Two solutions can be considered:
1. Individualize the notifications sent when a person participating in an event becomes a confirmed case (such as the notification sent in the event of a named assignment of a task)
2. Make a more substantial development allowing you to configure the choice of notifications that you want to activate or not. There will no longer be "other notifications" grouping together several actions, but the notifications will be individually configurable for each action.

### Additional Information
@ftavin @bernardsilenou @emond-fuseau

### PR DESCRIPTION
**Pull request by aponcon-atolcd**
Closes #4732

### CODE REVIEW

**Comment by aponcon-atolcd** [comment]
For information, this PR is not about making a global configuration file for notifications. It will be addressed later (on another PR).

File: sormas-api/src/main/java/de/symeda/sormas/api/feature/FeatureType.java [hunk 1]
```diff
@@ -52,6 +52,7 @@
 		new FeatureType[] {
 			TASK_MANAGEMENT }),
 	MANUAL_EXTERNAL_MESSAGES(true, true, null),
+	CONFIRMED_CASE_NOTIFICATIONS(true, true, null),
```
Comment (MateStrysewske) [inline sormas-api/src/main/java/de/symeda/sormas/api/feature/FeatureType.java:55]: Rename to EVENT_PARTICIPANT_CASE_CONFIRMED_NOTIFICATIONS (very long, I know, but the current name sounds like it's associated to ALL notifications sent out when a case gets confirmed)

File: SERVER_CUSTOMIZATION.md [hunk 2]
```diff
@@ -135,6 +135,9 @@ The following features are currently configurable:
 * **Line Listing** `LINE_LISTING`: Whether or not using line listing for case entry is enabled in the specified jurisdiction for the specified disease. Configurable from the UI, no database interaction needed.
 * **Documents** `DOCUMENTS`: Enables document storage.
 * **GDPR Popup** `GDPR_CONSENT_POPUP`: Enables a popup which informs new users about the General Data Protection Regulation (GDPR) of the European Union
+* **Task related notifications** `TASK_NOTIFICATIONS`: Enables email and SMS notifications related to tasks management
+* **Confirmed case related notifications** `CONFIRMED_CASE_NOTIFICATIONS`: Enables email and SMS notifications sent to the surveillance officer of events where a participant is a confirmed case
```
Comment (MateStrysewske) [inline SERVER_CUSTOMIZATION.md:139]: Change to:

``` 
**Notifications related to confirmed cases of event participants** `EVENT_PARTICIPANT_CASE_CONFIRMED_NOTIFICATIONS`: Enables email and SMS notifications sent to surveillance officers of events when the case of an event participant is confirmed
```

File: SERVER_CUSTOMIZATION.md [hunk 3]
```diff
@@ -135,6 +135,9 @@ The following features are currently configurable:
...
+* **Other notifications** `OTHER_NOTIFICATIONS`: Enables other email and SMS notifications (other than those impacted by `TASK_NOTIFICATIONS` and `CONFIRMED_CASE_NOTIFICATIONS` features)
```
Comment (MateStrysewske) [inline SERVER_CUSTOMIZATION.md:140]: CONFIRMED_CASE_NOTIFICATIONS => EVENT_PARTICIPANT_CASE_CONFIRMED_NOTIFICATIONS
Also, remove the "features" word at the end

**Review by MateStrysewske — CHANGES_REQUESTED** [review]
See comments above (forgot to click on Request changes :))

**Review by MateStrysewske — APPROVED** [review]

**Pull request merged**